Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -71,6 +72,29 @@ const CreateNewMaintenanceWindow = () => {
     }));
   };
 
+  const handleSubmit = async () => {
+    const data = {
+      repeat: values.repeat,
+      date: values.date.format("YYYY-MM-DD"),
+      startTime: values.startTime.format("HH:mm"),
+      duration: values.duration,
+      unit: values.unit,
+      friendlyName: values.friendlyName,
+      addMonitors: values.AddMonitors,
+    };
+
+    console.log("Submitting data: ", data);
+
+    await axios
+      .post("/address-of-the-route", data)
+      .then((res) => {
+        console.log("Response: ", res.data);
+      })
+      .catch((err) => {
+        console.log("Error: ", err);
+      });
+  };
```

Comment:
We can remove this for now, let's just marshal the data correctly first and make sure it is validated